src/schema/v2/sale_artwork.ts

```diff
@@ -315,7 +315,7 @@ export const SaleArtworkType = new GraphQLObjectType<any, ResolverContext>({
       },
       lotLabel: {
         type: GraphQLString,
-        resolve: ({ lot_label }) => lot_label,
+        resolve: ({ lot_label }) => lot_label?.match(/\d+/)?.shift(),
```

Comment:
Two things:

- I notice there [already is a `lot_number` field](https://github.com/artsy/gravity/blob/796e90e4492e5667c75907d54d5b3964e8e2c191/app/models/domain/sale_artwork.rb#L19) which appears to be the `lot_label` but [parsed](https://github.com/artsy/gravity/blob/796e90e4492e5667c75907d54d5b3964e8e2c191/app/models/domain/sale_artwork.rb) as an int 

- My  hesitation is that I think alphanumeric lot numbers may be legitimate (maybe even crucial?) in rare circumstances?

```rb

pp SaleArtwork.order(created_at: :desc).limit(1000).
  pluck(:lot_label, :lot_number).
  select{ |label, number| label != number.to_s }

[["79b", 79],
 ["79a", 79],
 ["256a", 256],
 ["281a", 281],
 ["257a", 257],
 ["93A", 93],
 ["221a", 221],
 ["203a", 203],
 ["190a", 190],
 ["182a", 182],
 ["171a", 171],
 ["217a", 217],
 ["269a", 269]]
```

Reply:
I don't really know how `lot_number` is used, maybe it's involved in sorting somewhere? (Though it would be dicey for that if `79a` and `79b` both become `79` 😅 )

**suggestion (non-blocking):** A trickier regex could work. But then I wonder if should be an option rather than a default, controlled via e.g. a `trim` argument? That way we could more gradually opt in to this behavior where it's needed, like the artwork screen redesign.

Reply:
Good thought on an optional argument. Sounds like a safer iteration than swapping in a new resolution for all clients right away.

Went ahead and pushed that change if you'd like to take a look 👍🏼